Shared/Samples/Choose camera controller/README.md

```diff
@@ -0,0 +1,31 @@
+# Choose camera controller
```

Comment:
Sorry I didn't spot this in the first place, can you change the title to `Change camera controller` to match the common design? Thanks. 🤦 

- the folder name
- README title
- title in metadata
- swift filename
- view type name
- screenshot name and filename